Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:latest
```

Comment:
Please add the usual file headers (Apache license etc.), similar to other files in scripts/.

Also, if you look at most of the Oppia scripts, there is a long descriptive comment at the top explaining how the file should be used. I would suggest that we do that here as well. Explain any limitations here, too.

Reply:
Will do later tonight. Thanks Sean

Reply:
Will have to do this and the isDocker explanations tomorrow!

Reply:
I think this is done -- in the future, please make sure to reply "Done" to a comment thread when it's done, and resolve it. Thanks!

Reply:
Yep. This is done. I'll make sure to do that!